docs/kinesis/kinesis_connector.md

```diff
@@ -0,0 +1,149 @@
+# Exasol Kinesis Connector 
+
+Exasol Kinesis Connector provides UDF scripts that allow users to import data
+from [Kinesis Streams][kinesis-streams] to an Exasol table.
+
+## Prerequisites
+
+* A running Exasol cluster with a version 6.0 or later.
+* An AWS account with all necessary permissions to read from a Kinesis Stream.
+* An AWS Kinesis producer which sends the data to a stream in the valid JSON
+ format. 
+ An example of a valid JSON string: 
+ `{"sensorId": 17,"currentTemperature": 147,"status": "WARN"}`.
+ An example of an invalid JSON format:
+ `{"sensorId": 17,"currentTemperature": 147,"status": "WARN",}`. The Exasol
+ Kinesis Connector will not parse this string correctly.
+
+## Deployment
+
+Please refer to the [deployment guide](../deployment_guide.md) till the section
+`Upload the JAR file to the bucket` (inclusive) to prepare a bucket and a JAR.
+
+## Preparing a Table for Data
+
+To store the data from a stream, we need a table. 
+The columns in the Exasol table have to imitate the data types of the data
+ stored in the stream and also be in the exact order.
+ 
+The table also requires two additional columns to store the Kinesis metadata:  
+    `KINESIS_SHARD_ID VARCHAR(2000), 
+     SHARD_SEQUENCE_NUMBER VARCHAR(2000)`
+These two columns must be at the end of the table.
+ 
+For example, given the following stream data, 
+`{"sensorId": 17,"currentTemperature": 147,"status": "WARN"}`, 
+the Exasol table should be as below:
+
+```sql
+CREATE SCHEMA <schema_name>;
+
+CREATE OR REPLACE TABLE <schema_name>.<table_name> 
+    (SENSORID DECIMAL(18,0), 
+    CURRENTTEMPERATURE DECIMAL(18,0), 
+    STATUS VARCHAR(100), 
+-- Required for importing data from Kinesis
+    KINESIS_SHARD_ID VARCHAR(130), 
+    SHARD_SEQUENCE_NUMBER VARCHAR(2000));
+
+```
+
+## Create ETL UDFs Scripts
+
+Create the following UDF scripts. Please do not change the names of the scripts.
+
+```sql
+--/
+CREATE OR REPLACE JAVA SET SCRIPT KINESIS_METADATA (...) 
+EMITS (KINESIS_SHARD_ID VARCHAR(130), SHARD_SEQUENCE_NUMBER VARCHAR(2000)) AS
+  %scriptclass com.exasol.cloudetl.kinesis.KinesisShardsMetadataReader;
+  %jar /buckets/bfsdefault/kinesis/cloud-storage-etl-udfs-<VERSION>.jar;
```

Comment:
You could have our `version` script inject the latest versions here automatically like in the Virtual Schemas. This would be more convenient for the end users.

Reply:
I think this will not work. The version is obtained using `git describe`. So every commit has a version, e.,g `tag-commitNumberSince-sha`. 

Also, I think it will be too much. User can obtain the latest released jar from github releases.